server/routes/trackReceipt.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const router = express.Router()
+const jwt = require('../utils/jwt')
+const TrackReceipt = require('../models/TrackReceipt')
+
+router.post('/', jwt.verifyJWT, async (req, res, next) => {
+  /* adds track receipt to db */
+  try {
+    const userId = req.userId
+    const { otherUserId, track, username, similarityScore, playlist, addedAt } = req.body
+
+    const receipt = new TrackReceipt({ userId, otherUserId, track, username, similarityScore, playlist, addedAt })
+    await receipt.save()
+    res.status(200).json()
+  } catch (error) {
+    next(error)
+  }
+})
+
+router.get('/', jwt.verifyJWT, async (req, res, next) => {
+  /* returns all track receipts belonging to current user */
+  try {
+    const userId = req.userId
+    const receipts = await TrackReceipt.find({ userId })
```

Comment:
Nit: singular because find returns a single object.

Reply:
This find returns an array of all the objects in the collection that has the specified userId.